code/game/objects/items/weapons/implants/implantfluff.dm

```diff
@@ -0,0 +1,112 @@
+	function = "Uses a mixture of cybernetic nanobots, bone regrowth chemicals, and radio signals to quickly break the user out of restrains"
+
+/datum/implant_fluff/health
+	name = "Nanotrasen Health Implant"
+
+/datum/implant_fluff/krav_maga
+	name = "Prospero Foreign Industries Krav Maga Neurotrainer"
+	life = "Will cease functioning 4 hours after death of host"
+	notes = "As a consequence of using a neurotrainer, using this implant will overwrite other martial arts knowledge in the users brain"
+	function = "Teaches even the clumsiest host the arts of Krav Maga."
+
+/datum/implant_fluff/mindshield
+	name = "Nanotrasen Type 3 Mindshield Implant"
+	life = "Studies have shown the implant to last up to 10 years"
+	notes = "This is the third iteration of an implant that has garnered attention from many galactic humanoid rights groups over concerns of self autonomy. Allegedly, it used to force the user to be completely loyal to Nanotrasen."
+	function = "Personnel injected with this device can better resist mental compulsions such as brainwashing and mindslaving."
+
+/datum/implant_fluff/storage
+	name = "Cybersun Industries RX-16 Collapsible Body Cavity Implant"
+	notes = "Prolonged usage of this implant often results in the users bones being fractured and occassionaly complete organ failure."
+	function = "Allows the user to store two small objects within a cybernetic body cavity."
+
+/datum/implant_fluff/tracking
+	name = "Nanotrasen RFID Tracking Chip"
+	life = "Unknown, known to last up to a few years"
+	notes = "Tracking implants are Neuro-safe! It uses Specialized shell that will melt and disintegrate into bio-safe elements if it ever malfunctions. "
+	function = "Continuously transmits low power signal. Can be used for tracking and teleporting to the user."
+
+/datum/implant_fluff/traitor
+	name = "Syndicate Type E Mindslave Implant"
+	life = "Unknown, no collected sample has been active long enough to determine lifespan"
+	notes = "Any humanoid injected with this implant will become loyal to the injector, unless of course the host is already loyal to someone else. It's Diplomacy made easy. The Syndicate is known to use this for political and diplomatic leverage."
+	function = " Contains a small pod of nanobots that manipulate the host's mental functions and slave the host to the implanter."
+
+/datum/implant_fluff/uplink
+	name = "Syndicate Agent Uplink Implant"
+	life = "Unknown, no collected sample has been active long enough to determine lifespan"
+	notes = "An intricate piece of machinery that interfaces directly with the users brain and which inner workings area a closely guarded syndicate secret."
+	function = "Allows the user to access a syndicate uplink connected contained within their body, invisible to the outside but provides the same functionality as a regular uplink."
+
+/datum/implant_fluff/sad_trombone
+	name = "BioTech Solution Comedy Implant"
```

Comment:
Some of these fluff messages say Biotech Solution, some of them say Biotech Solutions. Probably want to standardize it.

Reply:
fixed